README.rst

```diff
@@ -12,7 +12,7 @@ The motivation behind this is `a post <https://medium.com/@kentcdodds/first-time
 
 `N. Geisler <https://github.com/geislern>`_ added support for more labels: ``up-for-grabs``, ``beginners``, ``low-hanging-fruit``.
 
-See it in action at `@first_tmrs_only`_.
+See it in action at `@first_timers_only`_.
```

Comment:
corrected the spelling

Reply:
Twitter has limits on the length of handles. So I had chosen @first_tmrs_only as the handle.

The spelling is correct. :)